cosmo_tester/resources/attributes.yaml

```diff
@@ -23,24 +23,26 @@ cloudify_manager_4_0_image_name: cloudify-manager-premium-4.0
 cloudify_manager_3_4_2_image_name: manager3.4.2-insecure-image
 notmanager_image_name: CentOS-7-x86_64-GenericCloud-1608
 
-centos7_username: centos
-centos7_image_name: CentOS-7-x86_64-GenericCloud-1608
+centos_7_username: centos
```

Comment:
God bless you on all of these. I hated the lack of standardization

Reply:
Yeah, it was quite irritating- and it also meant that there were a bunch of places we needed a pile of different methods rather than just one that substituted in OS names.